sources/api/storewolf/src/main.rs

```diff
@@ -373,7 +298,14 @@ fn populate_default_datastore<P: AsRef<Path>>(
             .context(error::QueryData)?;
     } else {
         info!("Creating datastore at: {}", &live_path.display());
-        create_new_datastore(&base_path, version)?;
+        let version = match version {
+            Some(v) => v,
+            None => {
+                let br = BottlerocketRelease::new().context(error::ReleaseVersion)?;
+                br.version_id
+            }
+        };
+        create_new_datastore(&base_path, &version).context(error::DatastoreCreation)?;
```

Comment:
Do we have to move this `version` behavior from the function to `main`?  Can it stay in the function, so we keep things how they are, with the function being able to automatically determine the version if needed?

Reply:
Will restore the function's behavior. I haven't tested an instance, will do so.

Reply:
[b372001](https://github.com/bottlerocket-os/bottlerocket/compare/6c03e6d1db7ca40c127c444984e07bb8f2279920..b372001434b563e72db7f0d630d0d940c764933f)